aws/rust-runtime/aws-config/src/profile/credentials/exec.rs

```diff
@@ -21,18 +21,22 @@ use aws_smithy_client::erase::DynConnector;
 use aws_types::credentials::{self, CredentialsError, ProvideCredentials};
 
 use std::fmt::Debug;
+use tracing::Instrument;
+use crate::profile::mfa_token::ProvideMfaToken;
 
 #[derive(Debug)]
 pub struct AssumeRoleProvider {
     role_arn: String,
     external_id: Option<String>,
     session_name: Option<String>,
+    mfa_serial: Option<String>,
```

Comment:
There should be a way to set this from the `AssumeRoleProviderBuilder` as well for those that aren't using the profile credentials file. In there, the name should probably match the name in STS's AssumeRole operation, `serial_number`.

Reply:
Will do 👍

Reply:
Have pushed this